packaging/macosx/BUILD.txt

```diff
@@ -25,12 +25,18 @@ How to make disk image with darktable application bundle (64 bit Intel only):
     Install required dependencies:
      $ sudo port install git exiv2 libgphoto2 gtk-osx-application-gtk3 lensfun librsvg libsoup openexr json-glib flickcurl GraphicsMagick openjpeg lua webp libsecret pugixml osm-gps-map adwaita-icon-theme tango-icon-theme intltool
     To have working OpenMP (at least partly, only for C source files) you will also need gcc6:
+    (llvm port, which gets installed as dependency, won't compile with macosx_deployment_target set, so comment it out when llvm is built:)
+    Comment out 'macosx_deployment_target 10.7' that you added in file '/opt/local/etc/macports/macports.conf' (refer to line 6) then
+    $ port install llvm-5.0
+    $ port install llvm_select
+    Remove the comment # before 'macosx_deployment_target 10.7' in the file '/opt/local/etc/macports/macports.conf' and proceed:
      $ sudo port install gcc6
-    (llvm port, which gets installed as dependency, won't compile with macosx_deployment_target set, so comment it out when llvm is built)
     Clone darktable git repository (in this example into ~/src):
      $ mkdir ~/src
      $ cd ~/src
      $ git clone git://github.com/darktable-org/darktable.git
```

Comment:
Not a big deal, but could pass `--recursive` to `git clone` here - this would make the next two steps unnecessary.